pkg/crc/preflight/preflight_windows.go

```diff
@@ -63,55 +64,24 @@ var hypervPreflightChecks = []Check{
 			return nil
 		},
 		fixDescription: "Creating crc-users group",
-		fix: func() error {
-			if _, _, err := powershell.ExecuteAsAdmin("create crc-users group", "New-LocalGroup -Name crc-users"); err != nil {
-				return fmt.Errorf("failed to create 'crc-users' group: %v", err)
-			}
-			return nil
-		},
-		cleanupDescription: "Removing 'crc-users' group",
-		cleanup: func() error {
-			if _, _, err := powershell.ExecuteAsAdmin("remove crc-users group", "Remove-LocalGroup -Name crc-users"); err != nil {
-				return fmt.Errorf("failed to remove 'crc-users' group: %v", err)
-			}
-			return nil
-		},
-		labels: labels{Os: Windows},
+		flags:          NoFix | StartUpOnly,
```

Comment:
I'd prefer to only add `|StartUpOnly` in this commit, and remove the unused fix/cleanup in a follow-up commit, easier to review. This is only a slight preference, if you prefer to keep everything together, I'll do with it.